build.sbt

```diff
@@ -713,20 +713,62 @@ lazy val examples = (projectMatrix in file("examples"))
 
 val compileDocs: TaskKey[Unit] = taskKey[Unit]("Compiles docs module throwing away its output")
 compileDocs := {
-  (docs.jvm(scala2_12) / mdoc).toTask(" --out target/sttp-docs").value
+  (docs.jvm(scala2_13) / mdoc).toTask(" --out target/sttp-docs").value
 }
 
+val jeagerClientVersion = "1.0.0"
+val braveOpentracingVersion = "0.34.2"
+val zipkinSenderOkHttpVersion = "2.15.0"
+
 lazy val docs: ProjectMatrix = (projectMatrix in file("generated-docs")) // important: it must not be docs/
-  .settings(commonSettings)
-  .settings(publishArtifact := false, name := "docs")
-  .dependsOn(core)
   .enablePlugins(MdocPlugin)
+  .settings(commonSettings)
   .settings(
     mdocIn := file("docs"),
     moduleName := "sttp-docs",
     mdocVariables := Map(
-      "VERSION" -> version.value
+      "VERSION" -> version.value,
+      "JEAGER_CLIENT_VERSION" -> jeagerClientVersion,
+      "BRAVE_OPENTRACING_VERSION" -> braveOpentracingVersion,
+      "ZIPKIN_SENDER_OKHTTP_VERSION" -> zipkinSenderOkHttpVersion
     ),
-    mdocOut := file("generated-docs/out")
+    mdocOut := file("generated-docs/out"),
+    publishArtifact := false,
+    name := "docs",
+    libraryDependencies ++= Seq(
+      "org.json4s" %% "json4s-native" % json4sVersion,
+      "io.circe" %% "circe-generic" % "0.12.1",
+      "commons-io" % "commons-io" % "2.7",
+      "io.github.resilience4j" % "resilience4j-circuitbreaker" % "1.5.0",
+      "io.github.resilience4j" % "resilience4j-ratelimiter" % "1.5.0",
```

Comment:
extract version

Reply:
Any particular reason for doing that? Not all versions are extracted e.g. 
L431:
```
      "org.asynchttpclient" % "async-http-client" % "2.12.1"
```

I don't see any gains from that, only additional noise.